cmd/create/cluster/cmd.go

```diff
@@ -404,10 +412,31 @@ func init() {
 	)
 	flags.MarkHidden("properties")
 
+	flags.StringVar(
```

Comment:
I'm wondering if instead of manually adding all these flags you could do something like
```
	flags.AddFlagSet(operatorroles.Cmd.Flags())
```
And that would take care of adding the flags from the operator roles command, rather than duplicating code.

Reply:
This leads to a nil pointer as the operator flags also has a cluster field. Unsure if we want to change the flags for the operator role cmd. Would this have a knock on to docs etc?